packages/tanzu-auth/bundle/config/deployment.yaml

```diff
@@ -16,8 +16,13 @@ spec:
     #! TODO: check out the spec of other packages, such as addons-manager-v1 for resource limits, etc.
     #! we probably want to flesh this out.
     spec:
+      nodeSelector: #@ data.values.deployment.nodeSelector
       serviceAccountName: tanzu-auth-controller-manager-sa
       containers:
       - image: #@ data.values.controller.image
         name: tanzu-auth-controller-manager
         args: ["--disable-cascade-v1alpha1"]
```

Comment:
don't we need to do something with `--metrics-bind-addr=0` here?

Reply:
I can make it:
```
args: ["--disable-cascade-v1alpha1", "--metrics-bind-addr=0"]
```
Quick looking now to see if I can figure out why we opt to hard-code rather than provide via values.yaml.

Reply:
I guess we are good with just no metrics 👍

Reply:
:shrug: i guess. i had originally thought that `=0` was "listen on a random port" but i guess i am wrong about that